app/src/js/components/SortableTable/SortableTable.js

```diff
@@ -35,18 +35,16 @@ IndeterminateCheckbox.propTypes = {
 };
 
 const SortableTable = ({
-  sortId,
-  initialSortId,
-  rowId,
-  order = 'desc',
   canSelect,
   changeSortProps,
-  tableColumns = [],
+  clearSelected,
   data = [],
+  initialHiddenColumns = [],
   onSelect,
-  clearSelected,
+  rowId,
+  shouldManualSort = false,
   shouldUsePagination = false,
-  initialHiddenColumns = []
+  tableColumns = [],
```

Comment:
This looks like you removed sortId and order, in favor of always having a sortBy but I see sortId still used on <SortableTable’s.  execution-events.js   Is that an oversight, or does it get passed down somehow to another component?

Reply:
Yeah this was an oversight. The idea is that when using a `<SortableTable>` via `<List>`, we're doing all of the request for the data and sorting server side (aka manual sorting). The `initialSortId` prop on `<List>` allows us to set up the sort_key for the API request, but then all of the rest of the sorting will be handled based on the user's interaction with the table, so the props are never updated/used anyway.

On the other hand, when using a plain `<SortableTable>`, we're leveraging the react-table's automatic/internal sorting feature, so we also do not need to pass in props for that.